src/Recaptcha.js

```diff
@@ -21,6 +21,32 @@ export default {
     },
     tabindex: {
       type: String
+    },
+    loadRecaptchaScript: {
+      type: Boolean,
+      default: false
+    },
+    recaptchaScriptId: {
+      type: String,
+      default: '__RECAPTCHA_SCRIPT'
+    },
+    recaptchaHost: {
+      type: String,
+      default: 'www.google.com'
+    }
+  },
+  beforeMount () {
+    if (loadRecaptchaScript) {
```

Comment:
Shouldn't this be `this.loadRecaptchaScript`?

Reply:
Oops, my bad :penguin: